pulse2percept/utils.py

```diff
@@ -13,8 +16,27 @@
     has_jit = False
 
 
+def deprecated(func):
```

Comment:
Maybe add a key-word argument for "alt_func", which, if defined, will trigger a block that also prints a `"Use {} instead".format(alt_func.__name__)`?